src/multi.rs

```diff
@@ -982,6 +982,11 @@ impl<H> Easy2Handle<H> {
         self.easy.get_mut()
     }
 
+    /// Consumes the Easy2 handle, returning the underlying handler.
+    pub fn into_inner(self) -> H {
+        self.easy.into_inner()
+    }
+
```

Comment:
Maybe this should return an Easy instead? It wouldn't be symmetric with `new` however, and not sure if there's much value in it.